### ISSUE TITLE
Worklist for 29

### ISSUE THREAD

**Issue by GoogleCodeExporter** [classilla/tenfourfox#267]
```
1. Endian problem with icon images?
2. Snapshot thumbnails use ContentParent and crash
3. Snapshot thumbnails have a weird blue tint (probably also endian) (issue 236)
4. Issue 247
5. Fix background colour of windows
6. Remove spurious resize gadget at bottom of window bar
7. See if mAvailableFonts goes bad other places, probably fallout from 
disabling the deferred font loader

```

Original issue reported on code.google.com by `classi...@floodgap.com` on 24 Mar 2014 at 3:44
- Blocked on: #236
